app/components/documents/document-card.hbs

```diff
@@ -1,202 +1,200 @@
 <div
-  class="auk-u-mb-2
-    {{if this.documentContainer.aboutToDelete "vlc-document--deleted-state"}}"
+  class="vlc-document-card {{if this.documentContainer.aboutToDelete "vlc-document--deleted-state"}}"
```

Comment:
Git's diff is struggling with this one. What changed here: the wrapping `<div>` was removed (mostly because I'm fairly sure it's not needed), it's properties were placed in `<div [...] data-test-document-card>` and the CSS class `auk-u-mb-2` was removed. All the rest is just indentation changes.